bin/sam-translate.py

```diff
@@ -28,6 +26,9 @@
 import boto3
 from docopt import docopt
 
+if sys.version_info.major == 3:
```

Comment:
We can drop this version check and just import the module as we are already py3 only on the `develop` branch

Reply:
Removed.